docs/02_cleos/03_command-reference/net/status.md

```diff
@@ -1,31 +1,66 @@
+## Command
+```sh
+cleos net status [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to query status of connection.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-status of existing connection
+Returns the status of a connected peer. This command allows a node operator to check the status of a node's connected peer.
 
-**Command**
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos net status` command:
 
-```sh
-cleos net status
-```
-**Output**
+### Positionals
+* `host` _TEXT_ REQUIRED - The hostname:port to query status of connection.
```

Comment:
Is this one that doesn't have a full stop at the end?

Reply:
Removed trailing period. Indeed, parameter descriptions will mainly consist of non-full sentences, so a trailing period is not needed. But if these were full sentences, a period is required.